test/thresholds.service.spec.js

```diff
@@ -398,10 +404,18 @@ describe('thresholds service', function () {
   describe('getThresholdsFor', function () {
     it('takes an array of objects with location, allocations and plans fields and returns an object of location thresholds', function (done) {
       var stockCounts = [ // Note: it doesn't work yet with zones
+        { location: { national: 'national' }, date: { year: 2016, week: 2 } },
         { location: { zone: 'nc', state: 'kogi' }, date: { year: 2016, week: 2 } },
         { location: { zone: 'nc', state: 'kogi', lga: 'adavi' }, date: { year: 2016, week: 3 } }
       ]
       var expected = {
+        'national:national': {
```

Comment:
Could you make it use `national` as a key here instead of `national:national`? This is more consistent with how is it make with other locations in which the key is the name of the location doc.